geometryAbstractPractice/src/main/java/comp/fundacionjala/geometryAbstractPractice/Square.java

```diff
@@ -0,0 +1,26 @@
+package comp.fundacionjala.geometryAbstractPractice;
+
+public class Square extends Rectangle implements IShape {
+
+    public static final double SHAPE_SIDES=4;
+
+    public Square(double side) {
+        super(side);
+    }
+
+    public double calculateArea() {
```

Comment:
The method calculateArea and calculatePerimeter are not necessary
because they are inherited from Rectangle class